build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -186,8 +186,8 @@ rubygem-puppet-lint:
   :rpm_name: rubygem-puppet-lint-1.1.0-1.el7.noarch.rpm
   :source: http://yum.puppetlabs.com/el/7/dependencies/x86_64/rubygem-puppet-lint-1.1.0-1.el7.noarch.rpm
 rubygem-rake:
-  :rpm_name: rubygem-rake-0.9.6-29.el7.noarch.rpm
-  :source: http://mirrors.advancedhosters.com/centos/7.3.1611/os/x86_64/Packages/rubygem-rake-0.9.6-29.el7.noarch.rpm
+  :rpm_name: rubygem-rake-0.9.6-25.el7_1.noarch.rpm
+  :source: ftp://ftp.icm.edu.pl/vol/rzm6/linux-slc/centos/7.2.1511/os/x86_64/Packages/rubygem-rake-0.9.6-25.el7_1.noarch.rpm
```

Comment:
We need to stay away from `ftp` references in this file. Also, why did this package roll **backwards** in version?